package.json

```diff
@@ -67,6 +67,7 @@
     "@ava/typescript": "^1.1.1",
     "@babel/core": "^7.9.6",
     "@octokit/graphql": "2.1.1",
+    "@types/got": "^9.6.11",
```

Comment:
Are we using the `got` types anywhere? I don't see it in this PR. Also, fwiw, got^11 provides its own types, so I think it might be worth using that instead of this contributed types package

Reply:
Hmm, I thought there was an explicit reason for not using the newest `got`.    In either case, we're using a response from got on line 42 of tea.ts (const bucketMapList = <bucketMapResponse>JSON.parse(apiResponse.body);), so having the proper type seems relevant.  

Let me give got^11 a whirl.

Reply:
Seems fine: 5839e1276